pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -3054,3 +3334,14 @@ func enableFeatures(t *testing.T, features []string) func(context.Context) conte
 		return s.ToContext(ctx)
 	}
 }
+
+func getContextBasedOnFeatureFlag(featureFlag string) context.Context {
```

Comment:
you can use this helper https://github.com/tektoncd/pipeline/blob/main/pkg/apis/pipeline/v1beta1/pipelinerun_validation_test.go#L844-L855

Reply:
It seems like we cannot import `v1beta1_test` package from `pipeline_validation_test.go` that is under `v1beta1` package, can we?

Reply:
oh that's frustrating-- ideally pipeline_validation_test.go should be in the `v1beta1_test` package instead of `v1beta1`, because we want to only test exported functions. So you can either move pipeline_validation_test.go to `v1beta1_test` (may or may not work depending on if we are already testing unexported functions) or you could move the helper to the `v1beta1` package instead which should definitely work.